apiserver/plane/app/views/external/base.py

```
@@ -1,71 +1,165 @@
             return Response(
                 {"error": "Task is required"}, status=status.HTTP_400_BAD_REQUEST
             )
 
-        final_text = task + "\n" + prompt
-
-        client = OpenAI(api_key=OPENAI_API_KEY)
-
-        response = client.chat.completions.create(
-            model=GPT_ENGINE, messages=[{"role": "user", "content": final_text}]
-        )
+        text, error = get_llm_response(task, request.data.get("prompt", False), api_key, model, provider)
```

Ruff (0.8.2)

Line too long (105 > 88)

(E501)